jsk_topic_tools/scripts/tf_to_transform.py

```diff
@@ -37,10 +31,10 @@ def _publish_transform(self, event):
                 now,
                 timeout=rospy.Duration(self.duration)
                 )
-        except (tf.LookupException,
-                tf.ConnectivityException,
-                tf.ExtrapolationException):
-            rospy.logerr('cannot get transform')
+        except Exception:
```

Comment:
I explain why I needed this change. 
Catching the Exception is not ideal solution.